src/js/utils/readOnly.js

```diff
@@ -0,0 +1,18 @@
+import { css } from 'styled-components';
+
+export const readOnlyStyle = (theme) => {
+  const styles = [];
+  if (theme.global.input.readOnly?.border?.color)
+    styles.push(
+      css`
+        border-color: ${theme.global.input.readOnly.border.color};
+      `,
+    );
+  if (theme.global.input.readOnly?.background)
+    styles.push(
+      css`
+        background-color: ${theme.global.input.readOnly.background};
```

Comment:
```suggestion
        background: ${theme.global.input.readOnly.background};
```
Is there a case where this isn't a color and we'd want to be slightly more general with the CSS?

Reply:
good note, I changed this to be `background` instead of `background-color` just to allow a little more flexibility